AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_BasicTests.py

```diff
@@ -0,0 +1,135 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+class Tests:
+    open_existing_asset = (
+        "Existing material asset file opened successfully.",
+        "P0: Failed to open existing material asset file.")
+    close_opened_asset = (
+        "Closed existing material asset file.",
+        "P0: Failed to close existing material asset file.")
+    close_all_opened_assets = (
+        "Closed all currently opened asset document files.",
+        "P0: Failed to close all currently opened asset document files.")
+    close_all_opened_assets_except_one = (
+        "Closed all currently opened asset document files except for one file.",
+        "P0: Failed to close all currently opened asset document files except for one file.")
+    inspector_pane_visible = (
+        "Inspector pane is visible, MaterialEditor launch succeeded",
+        "P0: Inspector pane not visible, MaterialEditor launch failed")
+    changed_material_asset_color = (
+        "Material asset color changed successfully.",
+        "P0: Failed to change the color values of a material asset file.")
+    reverted_material_asset_color_change = (
+        "Material asset color reverted back to its initial color successfully.",
+        "P0: Failed to revert material asset color back to its initial color.")
+
+
+def MaterialEditor_BasicTests_ConfirmsBasicFunctionality():
```

Comment:
This name doesn't do much to help describe what is being tested, though it does hint at why.

Reply:
Its true but I couldn't come up with a better one for this basic smoke test set of tests. I'm going to push it through for now but will be more mindful of this in future updates.